code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -208,3 +208,40 @@ Difficulty: Medium
 	playsound(user, 'sound/magic/staff_change.ogg', 200, 0)
 	A.telegraph()
 	storm_cooldown = world.time + 200
+
+//Crown and Jewel of the Wastes
+/obj/item/clothing/ears/jewel_of_the_wastes //Takes headset but makes you have ear protection + same faction as the simple mobs
+	name = "jewel of the wastes"
+	desc = "A large blood red gem with metal clasps on it side. An echoing scream can be heard if placed next to the ear"
+	w_class = WEIGHT_CLASS_TINY
+	slot_flags = ITEM_SLOT_EARS
+	icon_state = "jewel_of_wastes"
+	item_state = "jewel_of_wastes"
+	icon = 'icons/obj/lavaland/artefacts.dmi'
```

Comment:
```suggestion
	icon = 'icons/obj/lavaland/artifacts.dmi'
```
Should be spelt this way, but depends if you want to spell check this on multiple items/the dmi file this is affected by.

Reply:
UK/US spelling differencies are non-issues, as stated on the contribution guidelines.

Reply:
Wasn’t aware it was a uk spelling, never seen it spelt that way.

Reply:
its how the file itself is spelled